quickstarts/managing.md

```diff
@@ -0,0 +1,52 @@
+# Managing a Terra environment
+
+## Logs
+The logs for a Terra environment created with this repo can be found in [Stackdriver](https://console.cloud.google.com/logs/viewer).
+
+Stackdriver has a [fairly robust querying syntax](https://cloud.google.com/logging/docs/view/advanced-queries) for sorting through logs. For example, since for a lot of terra services there are multiple instances of them and it can be nice to see logs from all instances of a specific service, the following advanced log query syntax aggregates together the logs from all SAM instances:
+```labels."compute.googleapis.com/resource_name":"sam-"```
```

Comment:
would be nice to also include an example of how to search for a string within all instance logs, I think it's like 

```
resource.type="gce_instance"
textPayload:"firestore"
```

Reply:
Good point, I'll add it!